packages/vrm-viewer/package.json

```
@@ -0,0 +1,68 @@
+{
+  "name": "@pixiv/vrm-viewer",
+  "version": "2.1.1",
+  "description": "The <vrm-viewer> web component that displays an interactive VRM model viewer",
+  "license": "MIT",
+  "author": "pixiv",
+  "files": [
+    "/lib/",
+    "/ts*/",
+    "/types/",
+    "LICENSE"
+  ],
+  "type": "module",
+  "main": "lib/vrm-viewer.module.js",
+  "types": "types/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./types-shim/index.d.ts",
+      "import": "./lib/vrm-viewer.module.min.js",
+      "require": "./lib/vrm-viewer.min.cjs"
```

```suggestion
      "import": "./lib/vrm-viewer.module.js",
      "require": "./lib/vrm-viewer.cjs"
```

See: https://github.com/pixiv/three-vrm/pull/1416